src/main/main.cpp

```diff
@@ -302,7 +313,7 @@ main(int argc, char* const* argv)
         Logging::setLogLevel(logLevel, nullptr);
 
         cfg.REBUILD_DB = newDB;
-        cfg.FORCE_SCP = forceSCP;
+        cfg.FORCE_SCP = (forceSCP != nullptr);
```

Comment:
cfg.FORCE_SCP is used in the code with the value true when the db state is force_scp==true.

I guess this line is just bogus: cfg.FORCE_SCP should be left as "false" until it's forced to "true" when reading from the database